source/boards/work-with-cards.rst

```diff
@@ -26,9 +26,9 @@ To add a new content block in the description section, hover over the section an
 - **Text**: Adds a new text block that can be formatted with Markdown.
 - **Image**: Select and embed an image file into the content block. The following image formats are currently supported: GIF, JPEG, and PNG.
 - **Divider**: Adds a divider content block below the previous block.
-- **Checkbox**: Adds a checkbox content block. Press Enter/Return after typing in content for your checkbox to add another checkbox within the same block. Please note, Markdown formatting is not supported within the *Checkbox* content block.
+- **Checkbox**: Adds a checkbox content block. Press Enter/Return after typing in content for your checkbox to add another checkbox within the same block. Please note, Markdown formatting isn't supported within the **Checkbox** content block.
 
-To manage the description content blocks on a card, hover over any existing block and select options menu **(...)** to move the block up or down, insert a new block above, or delete the current block. Alternatively, you can hover over any existing block, then click and hold the grid button to drag and drop it to a new position within the description section.
+To manage the description content blocks on a card, hover over any existing block and select options menu **(...)** to move the block up or down, insert a new block above, or delete the current block. Alternatively, you can hover over any existing block, then select and hold the grid button to drag and drop it to a new position within the description section.
 
 Card properties
 ---------------
```

Comment:
@justinegeffen Can we change line 57 under `Card properties` to:

- **Person** and **Multi-person** provides a quick way to capture user assignments. Note that this is not available in Personal Desktop.